src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -53,6 +53,9 @@
     <Compile Include="System\IO\FileLoadException.Interop.cs" />
     <Compile Include="System\IO\PathTooLongException.cs" />
     <Compile Include="System\IO\PathTooLongException.Interop.cs" />
+    <Compile Include="System\Uri\UriTests.CreateString.cs" />
+    <Compile Include="System\Uri\UriTests.CreateUri.cs" />
+    <Compile Include="System\Uri\UriTests.Methods.cs" />
```

Comment:
Can you move these 3 files out of the the 'Uri' subdirectory you made and in the 'System' directory?  We place files into directories/subdirectories based on the namespace of the type.  The 'System.Uri' class is in the 'System' namespace. Thx.

In terms of the filenames of the test source files, i.e. 'UriTests.CreateString.cs', it is probably better to rename those to something like 'Uri.CreateString.cs'.  This will match the rest of the set of test source files in this project.